presentation-api/cast.html

```diff
@@ -17,97 +17,86 @@ <h3>Background</h3>
   <code>start()</code> method on the <code>PresentationRequest</code> object.
 </p>
 
-{% capture initial_output_content %}
 <p>
-  <button id="createRequest">new PresentationRequest()</button>
+  <button id="start">presentationRequest.start()</button>
 </p>
 <p>
-  <button id="start" disabled>presentationRequest.start()</button>
+  <button id="close">presentationConnection.close()</button>
+  <button id="terminate">presentationConnection.terminate()</button>
 </p>
 <p>
-  <input id="presentationId" type="text" placeholder="presentationId">
-  <button id="reconnect" disabled>presentationRequest.reconnect(presentationId)</button>
+  <input id="presentationId" type="text" placeholder="Enter a presentation ID...">
+  <button id="reconnect">presentationRequest.reconnect(presentationId)</button>
 </p>
-<p>
-  <button id="close" disabled>presentationRequest.close()</button>
-  <button id="terminate" disabled>presentationRequest.terminate()</button>
-</p>
-{% endcapture %}
 {% include output_helper.html initial_output_content=initial_output_content %}
 
+<script>
+  log = ChromeSamples.log;
+</script>
+
 {% capture js %}
 // App ID EF1A139F is registered in the Google Cast SDK Developer
 // Console and points to the following custom receiver:
 // https://googlechrome.github.io/samples/presentation-api/receiver/index.html
-var presentationUrl = 'cast:EF1A139F';
-var presentationRequest;
-var presentationConnection;
+const presentationRequest = new PresentationRequest('cast:EF1A139F');
```

Comment:
Can you set navigator.presentation.defaultRequest here, so this presentation is the default choice when casting from the toolbar/menu?

Reply:
Done.